packages/locale/src/format/cldr/CldrZoneTextPrinterParser.js

```diff
@@ -61,6 +95,7 @@ export default class CldrZoneTextPrinterParser {
                     if (metaZoneData) {
                         return metaZoneData;
                     } else {
+                        const mapZones = getOrCreateMapZones(cldr);
```

Comment:
Instead of computing and passing `mapZones` in everytime, it's now only computed if we get to the part of the function where it's actually needed.